src/Spout/Writer/XLSX/Internal/Worksheet.php

```diff
@@ -146,7 +146,11 @@ public function addRow($dataRow, $style)
                     $cellXML .= ' t="s"><v>' . $sharedStringId . '</v></c>';
                 }
             } else if (CellHelper::isBoolean($cellValue)) {
-                $cellXML .= ' t="b"><v>' . $cellValue . '</v></c>';
+                if ($cellValue === true) {
```

Comment:
this can be simplified using `intval`:
`$cellXML .= ' t="b"><v>' . intval($cellValue) . '</v></c>';`